optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/ScoreDirectorFactoryFactory.java

```diff
@@ -309,7 +312,9 @@ private void validateNoDroolsAlphaNetworkCompilation() {
                         kieHelper.addResource(new FileSystemResource(scoreDrlFile));
                     }
                 }
-                kieBase = kieHelper.build(ExecutableModelProject.class, KieBaseMutabilityOption.DISABLED);
+                KieBaseConfiguration kieBaseConfiguration = buildKieBaseConfiguration(KieServices.get());
+                kieBaseConfiguration.setOption(KieBaseMutabilityOption.DISABLED); // Performance improvement.
+                kieBase = kieHelper.build(ExecutableModelProject.class, kieBaseConfiguration);
```

Comment:
Cannot resolve method 'build(java.lang.Class<org.drools.modelcompiler.ExecutableModelProject>, org.kie.api.KieBaseConfiguration)'

Does this PR depend on some latest changes in Drools?

Reply:
Yes, merged into Kogito today. Will be available tomorrow.